file-server/app.py

```diff
@@ -0,0 +1,149 @@
+import os
+import json
+from dotenv import load_dotenv
+from flask import Flask, request, jsonify
+from flask_cors import CORS
+from imgget import search_images, download_images
+import voiceovergen as vo
+import moviegen as mg
+import llm_req as llm
+import userstuff as us
+
+load_dotenv()
+
+PORT = os.environ["FILE_SERVER_PORT"]
+app = Flask(__name__)
+CORS(app)
+PALM_API_KEY = os.environ["PALMAPI"]
+
+
+@app.route("/text", methods=["POST"])
+def text():
+    """Send prompt to palm 2 API, get text. Use keywords to get images."""
+    if request.method == "POST":
+        user_id = us.generate_uuid()
+        us.generate_folder_structure(user_id)
+        data = json.loads(request.data.decode("utf-8"))
+        keywords = data["keywords"]
+        img_urls = search_images(os.environ["PEXELSAPI"], keywords)
+        download_images(img_urls, f"static/{user_id}/images")
+        imgs = os.listdir(f"./static/{user_id}/images")
+        txt = llm.generate_text(
+            "Write a 1 minute info speech in plaintext for :" + data["text"]
+        )
+        return jsonify(
+            {
+                "user_id": user_id,
+                "text": txt,
+                "image_urls": [f"/static/{user_id}/images/{image}" for image in imgs],
+            }
+        )
+
+
+@app.route("/edittext", methods=["POST"])
```

Comment:
why not just use @app.post, @app.get instead of using route and then having an `if` to verify request method? Applies to all routes btw

Reply:
Removing the secondary request checks